src/components/HeroSection/HeroSection.js

```diff
@@ -1,70 +1,157 @@
 import styled from "styled-components";
-import vector from "../../Images/Vector.svg";
+import { IoLogoJavascript, IoLogoHtml5 } from "react-icons/io";
+import { FaGraduationCap, FaReact } from "react-icons/fa";
 
 export const HeroContainer = styled.div`
   padding-left: 5%;
-  display: flex;
-  flex-direction: column;
-  margin-bottom: 3px;
-  background-image: url(${vector});
-  background-repeat: no-repeat;
-  background-position-x: right;
-  background-position-y: 100%;
+  width: 100%;
 `;
 
-export const ButtonContainer = styled.div``;
+export const ButtonContainer = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  width: 18%;
+  padding-top: 2%;
+  justify-content: space-between;
+
+  @media screen and (max-width: 780px) {
+    width: fit-content;
+  }
+`;
 
-export const Button = styled.button`
+export const Button = styled.a`
   background: ${(props) => (props.primary ? "#EE5CE6" : "transparent")};
```

Comment:
Acho que seria interessante configurar essas cores dentro do tema do Styled Components, aí ao invés de digitar as cores diretamente, como "#EE5CEG", você pode pegar do tema, e caso a mesma cor seja usado em vários lugares diferentes fica bem fácil de trocar se for necessário. Vi que o projeto tem dois temas: https://github.com/karenmscg/Portif-lio/blob/master/src/components/DarkMode/index.js. Aí talvez você precisaria salvar as cores nos dois se forem diferentes pra cada um.